imagepuller/internal/service/verify_test.go

```diff
@@ -214,6 +221,11 @@ func TestStoreAndVerifyLayers(t *testing.T) {
 			_, err = s.storeAndVerifyLayers(log, tc.stubImg)
 
 			assert.ErrorIs(err, tc.wantErr)
+
+			out := buf.String()
+			if tc.cachedLayerID != "" {
+				assert.Contains(out, tc.cachedLayerID)
+			}
```

Comment:
This check seems a bit brittle. Can't we count the number of times the stub has been called and work with that instead?

Reply:
Good idea, will do.